// Copyright 2018  The Cockroach Authors.
//
// Licensed as a Cockroach Enterprise file under the ZNBase Community
// License (the "License"); you may not use this file except in compliance with
// the License. You may obtain a copy of the License at
//
//     https://github.com/znbasedb/znbase/blob/master/licenses/ICL.txt

syntax = "proto3";
package znbase.icl.storageicl.engineicl.enginepbicl;
option go_package = "enginepbicl";

import "icl/storageicl/engineicl/enginepbicl/key_registry.proto";

// EncryptionStatus contains encryption-related information.
message EncryptionStatus {
  // Information about the active store key, if any.
  KeyInfo active_store_key = 1;
  // Information about the active data key, if any.
  KeyInfo active_data_key = 2;
}
